styles.css

```diff
@@ -2,12 +2,678 @@ body,
+  padding-top: 5%;
+  padding-bottom: 5%;
+  padding-left: 10%;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-top: 1px solid rgb(173, 171, 171);
+  border-right: 1px solid rgb(173, 171, 171);
+  border-bottom: 1px solid rgb(173, 171, 171);
+  border-left: 1px solid rgb(173, 171, 171);
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+#getting-started-form > .form-message > div:nth-child(3) {
+  grid-column: 1 / 3;
+  grid-row: 2;
+  margin-top: 1.5vh;
+  margin-right: 0;
+  padding-left: 2%;
+  padding-right: 0;
+}
+
+.yours textarea {
+  height: 100px;
+}
+
+#getting-started-form > .form-message > div:nth-child(3) > textarea {
+  margin: 0;
+}
+
+#getting-started-form {
+  text-align: center;
+}
+
+::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(29, 28, 28);
+  text-align: start;  /* Firefox */
+}
+
+::-ms-input-placeholder { /* Microsoft Edge */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(0, 0, 0);
+  text-align: start;
+}
+
+:-ms-input-placeholder { /* Internet Explorer 10-11 */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(0, 0, 0);
+  text-align: start;
+}
+
+.button-get-started {
+  width: 50%;
+  background: #1a2236;
+  color: #fff;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  border: 1px hidden;
+  margin-top: 5%;
+  margin-bottom: 50px;
+}
+
+.button-get-started:hover {
+  background: #fff;
+  color: #1a2236;
+  font-weight: 500;
+  border: solid 1px #1a2236;
+}
+
+.button-get-started:active {
+  background: #1a2236;
+  color: #fff;
+}
+
+.footer {
+  display: flex;
+  background: #1a2236;
+  color: #98dede;
+  align-items: center;
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
+
+.footer h2 {
+  margin: 0;
+  padding-left: 10vw;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.icon-sosmed {
+  display: flex;
+  flex-direction: row;
+  gap: 5vw;
+  width: 24px;
+  height: 24px;
+  padding-left: 20vw;
+}
+
+/* start about section desktop version */
+
+/* .all-about {
+  display: grid;
+  grid-template-columns: 55% 45%;
+  grid-template-rows: auto;
+}
+
+.works-lfs-grid {
+  grid-column: 1;
+  grid-row: 1;
+  display: grid;
+  grid-template-columns: 5% 1fr 1fr 3%%;
+  grid-template-rows: auto;
+  color: #fff;
+}
+
+.languages-section {
+  background: #1c528a;
+  grid-column: 2;
+  grid-row: 1;
+}
+
+.frameworks-section {
+  background: #e2505c;
+  grid-column: 3;
+  grid-row: 1;
+}
+
+.skills-section {
+  background: #febf52;
+  grid-column: 2;
+  grid-row: 2;
+}
+
+.about-section {
+  grid-column: 2;
+  grid-row: 1;
 } */
 
-@media only screen and (min-width: 375px) {
+/* end of  about section
+
+/* mobile version */
+@media screen and (max-device-width: 767px),
+  screen and (max-width: 767px) {
```

Comment:
As the previous reviewer mentioned, please use min-width value only since we follow mobile first approach. In this way, you will use less code for different screen sizes and your web pages will be more adaptive.

Reply:
changes to mobile first approach